srcpkgs/CalculiX-cgx/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'CalculiX-cgx'
+pkgname=CalculiX-cgx
+version=2.20
+revision=1
+wrksrc=CalculiX
+_cgx_dir=cgx_${version}
+build_style="gnu-makefile"
+make_build_args="-C ${_cgx_dir}/src"
```

Comment:
```suggestion
```
I ended up overriding all the useful parts of the build style so I don't think it's needed

Reply:
I tried to keep the build-style to "gnu-makefile" instead of overriding do_build(), and it seems to work. Is there a particular reason why you used a custom do_build() function?